usaspending_api/recipient/delta_models/sam_recipient.py

```diff
@@ -1,29 +1,107 @@
 SAM_RECIPIENT_COLUMNS = {
-    "awardee_or_recipient_uniqu": "STRING",
+    "id": "LONG NOT NULL",
+    "recipient_hash": "STRING",
     "legal_business_name": "STRING",
-    "ultimate_parent_unique_ide": "STRING",
-    "ultimate_parent_legal_enti": "STRING",
-    "broker_duns_id": "INTEGER NOT NULL",
-    "update_date": "DATE NOT NULL",
+    "duns": "STRING",
     "address_line_1": "STRING",
     "address_line_2": "STRING",
+    "business_types_codes": "ARRAY<STRING>",
     "city": "STRING",
     "congressional_district": "STRING",
     "country_code": "STRING",
+    "parent_duns": "STRING",
+    "parent_legal_business_name": "STRING",
     "state": "STRING",
-    "zip": "STRING",
     "zip4": "STRING",
-    "business_types_codes": "ARRAY<STRING>",
-    "dba_name": "STRING",
-    "entity_structure": "STRING",
+    "zip5": "STRING",
+    "alternate_names": "ARRAY<STRING>",
+    "source": "STRING NOT NULL",
+    "update_date": "TIMESTAMP NOT NULL",
     "uei": "STRING",
-    "ultimate_parent_uei": "STRING",
+    "parent_uei": "STRING",
 }
 
-sam_recipient_sql_string = rf"""
+sam_recipient_create_sql_string = fr"""
     CREATE OR REPLACE TABLE {{DESTINATION_TABLE}} (
         {", ".join([f'{key} {val}' for key, val in SAM_RECIPIENT_COLUMNS.items()])}
     )
     USING DELTA
     LOCATION 's3a://{{SPARK_S3_BUCKET}}/{{DELTA_LAKE_S3_PATH}}/{{DESTINATION_DATABASE}}/{{DESTINATION_TABLE}}'
     """
+sam_recipient_load_sql_string = fr"""
+-- Create temp table to reload into the duns table without dropping the destination table
+DROP TABLE IF EXISTS int.sam_recipient;
+
+CREATE TABLE int.sam_recipient AS (
+  SELECT
+    broker_SR.awardee_or_recipient_uniqu AS awardee_or_recipient_uniqu,
+    broker_SR.legal_business_name AS legal_business_name,
+    broker_SR.ultimate_parent_unique_ide AS ultimate_parent_unique_ide,
+    broker_SR.ultimate_parent_legal_enti AS ultimate_parent_legal_enti,
+    broker_SR.sam_recipient_id AS broker_duns_id,
+    broker_SR.record_date AS update_date,
+    broker_SR.address_line_1 AS address_line_1,
+    broker_SR.address_line_2 AS address_line_2,
+    broker_SR.city AS city,
+    broker_SR.congressional_district AS congressional_district,
+    broker_SR.country_code AS country_code,
+    broker_SR.state AS state,
+    broker_SR.zip AS zip,
+    broker_SR.zip4 AS zip4,
+    broker_SR.business_types_codes AS business_types_codes,
+    broker_SR.dba_name as dba_name,
+    broker_SR.entity_structure as entity_structure,
+    broker_SR.uei AS uei,
+    broker_SR.ultimate_parent_uei AS ultimate_parent_uei
+  FROM
+    global_temp.broker_SR as broker_SR
```

Comment:
Where did `broker_SR` get added as a `global_temp` (global temporary view) view? I don't see that in the PR code. Make sure to bring in latest changes from `qat` branch, as the ref data code moved to `spark.py`.
It should have a more clear name, like `broker_sam_recipient` rather than an abbreviation.